JavaScript/JavaScript1/Week3/exercise1-3.js

```diff
@@ -0,0 +1,84 @@
+// Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
```

Comment:
What happens if "Ahmad" is written in lowercase? What would happen if there is provided a name that is not in the list? Then `indexOf` will still work, but will remove the last item, which won't be the intended behavior. You could improve the code to cover those cases.